### PR TITLE
Andrew 3

### PR DESCRIPTION
**Pull request by athankso60**

### CODE REVIEW

File: models/company.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    required: false,
+  },
+  location: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: location,
+    required: true,
+  },
+});
+
+module.exports = mongoose.model('company', companySchema);
```
Comment (yas228) [inline models/company.js:20]: Use capital letters, applicable to all other schemas

Comment (athankso60) [reply models/company.js:20]: Should I use capitals also when referencing it?

Comment (yas228) [reply models/company.js:20]: yes

Comment (athankso60) [reply models/company.js]: done

**Pull request merged**